.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
Is this a new requirement of the Dart SDK or something that's likely to change before the final release?